### PR TITLE
revamp FAQ page

### PR DESCRIPTION
**Pull request by Joshument**
old FAQ page pertains to PolyMC, not Prism Launcher. This PR fixes that so that it pertains to what happened that caused **this** fork to be made (not PolyMC).

### CODE REVIEW

**Comment by Amateurintheflesh** [comment]
Prism Launcher was forked from PolyMC due to certain commits & executive decisions made by a maliciously inclined individual.

Prism Launcher was forked from PolyMC due to a sequence of actions by Lenny McLennington on the 17th of October, 2022. Lenny made several politically charged changes to the PolyMC GitHub repository and organization to remove those in support of “left-wing ideologies.” In a commit, Lenny removed the PolyMC code of conduct, which outlined and prohibited discrimination against the LGBTQIA+ community and subsequently removed all maintainers. The commit and subsequent mass purge of maintainers were done without any prior warning or discussion with the rest of the PolyMC team; the incident can be considered a hostile takeover.

Development of Prism Launcher requires community members & contributors to adhere to our [Code of Conduct](https://github.com/PrismLauncher/PrismLauncher/blob/develop/CODE_OF_CONDUCT.md). We do not agree with Lenny's actions, and thus have created **Prism Launcher** as a way to prevent any further politically inspired damage. Locking maintainers out of the project sets an unfavourable precedent, and does not ensure that the PolyMC project will remain in good hands in the future. We sincerely hope that you enjoy using Prism Launcher as we strive to deliver a consistent experience to our users.

### What does this mean for the future? Is PolyMC safe to use?

Prism Launcher will continue to have the same functionality as PolyMC did prior to [this malicious commit](https://github.com/PolyMC/PolyMC/commit/ccf282593dcdbe189c99b81b8bc90cb203aed3ee) and we will continue to add features from there. All contributors who were removed from the project have begun developing Prism Launcher, and as an end user switching to Prism Launcher will simply manifest as a change in logo & name for the time being.

We cannot guarantee that new changes to PolyMC will be non-malicious, and we advise that you migrate to Prism Launcher instead of continuing to risk using PolyMC. It is up to you whether or not you wish to continue using PolyMC, but Prism Launcher's Code of Conduct assures development will always occur in a fair manner that does not involve political motivation.

**Comment by Regular-Baf** [comment]
This would be better as a news post about the whole thing perhaps?

**Comment by Joshument** [comment]
I chose the FAQ method as PolyMC also used an FAQ to explain the fork.

**Comment by Su386yt** [comment]
```
# Frequently Asked Questions

## What happened to PolyMC? Why is there a new fork?

### TL;DR

Prism Launcher was forked from PolyMC due to specifc commits & executive decisions made by a maliciously inclined individual.

### What Happened?

Prism Launcher was forked from PolyMC due to a sequence of actions by Lenny McLennington on the 17th of October, 2022. Lenny made several politically charged changes to the PolyMC GitHub repository and organization to remove those supporting "left-wing ideologies". In a commit, Lenny removed the PolyMC Code of Conduct, which outlined and prohibited discrimination against the LGBTQIA+ community and subsequently removed all maintainers. The commit and subsequent mass purge of maintainers were done without any prior warning or discussion with the rest of the PolyMC team; the incident can be considered a hostile takeover.

Development of Prism Launcher requires community members & contributors to adhere to our [Code of Conduct](https://github.com/PrismLauncher/PrismLauncher/blob/develop/CODE_OF_CONDUCT.md). We disagree with Lenny's actions and have created **Prism Launcher** to prevent any further politically inspired damage. Locking maintainers out of a project sets an unfavourable precedent and does not ensure that the PolyMC project will remain in good hands. We sincerely hope that you enjoy using Prism Launcher as we strive to deliver a consistent experience to our users.

### What does this mean for the future? Is PolyMC safe to use?

Prism Launcher will continue to have the same functionality as PolyMC did before [this malicious commit](https://github.com/PolyMC/PolyMC/commit/ccf282593dcdbe189c99b81b8bc90cb203aed3ee) and we will continue to add features from there. All contributors who were removed from the project have begun developing Prism Launcher. As an end-user, switching to Prism Launcher will manifest as a change in logo & name for the time being.

We cannot guarantee that new changes to PolyMC will be non-malicious, and we advise that you migrate to Prism Launcher instead of continuing to risk using PolyMC. It is up to you whether or not you wish to continue using PolyMC. Still, Prism Launcher's Code of Conduct assures development will always occur in a fair manner that does not involve political motivation.

```

**Comment by Regular-Baf** [comment]
Maybe we should combine efforts on the FAQ page?
https://github.com/PrismLauncher/prismlauncher.github.io/pull/15
I have this PR from a few hours ago with my take on how the whole thing should look. Maybe we could incorporate some of the stuff here in there?

**Comment by Su386yt** [comment]
[faq.md](https://github.com/PrismLauncher/prismlauncher.github.io/files/9816807/faq.md)

**Comment by Joshument** [comment]
> Maybe we should combine efforts on the FAQ page? #15 I have this PR from a few hours ago with my take on how the whole thing should look. Maybe we could incorporate some of the stuff here in there?

Ah, sorry for not seeing that PR. If you would like to add to this one, you can comment the proposed changes and I'll sort them out when I wake up. I didn't mean to overshadow your PR. Alternatively just make a new commit on your PR, I don't mind either way.

**Comment by Su386yt** [comment]
I merged some of the changes with #15 

[faq.md](https://github.com/PrismLauncher/prismlauncher.github.io/files/9823539/faq.md)

**Comment by Regular-Baf** [comment]
I think we should avoid naming names entirely. Or dwelling too much on the situation. I'll have a go at combining the two now.

**Comment by Joshument** [comment]
> I merged some of the changes with #15
> 
> [faq.md](https://github.com/PrismLauncher/prismlauncher.github.io/files/9823539/faq.md)

I've made a few changes as I think that a lot of the PolyMC related info could've been either moved around or just removed entirely. If people disagree with this I will be more than happy to add them back.

> I think we should avoid naming names entirely. Or dwelling too much on the situation. I'll have a go at combining the two now.
If you can make it work, go for it, however we did come to a consensus in the discord that it was okay to name Lenny. Very open to other opinions after the fact though, as most of the discussion was during a few hours.
